source/iostreams.tex

```diff
@@ -12823,7 +12823,7 @@
 
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{pathobject = p},
+\effects Equivalent to: \tcode{pathobject = p},
```

Comment:
Do we want colons here? What follows is an expression, not a statement, non?

Reply:
No, it's not just the expression: all of what follows is part of the "Equivalent to". It seemed prudent to add the colon to highlight that fact.

Reply:
Hm, I thought it reads nicer without the colon, like a normal sentence. @zygoloid, a native speaker's opinion?

Reply:
Why do we add the colon for the "statement" case, then? I thought because "something big" was coming along.

Reply:
I always thought it was because of the grammatical nature of what follows: the colon separates sentences, or "statements", if you will. If you just have an expression, it's naturally an object of the current sentence.

Reply:
(I'm happy to merge everything except these two, which I'd like discussed further.)

Reply:
I've dropped these two contentious changes.